src/parameters/update_parameters.jl

```diff
@@ -56,6 +56,37 @@ function update_parameter_values!(
     end
 end
 
+function update_parameter_values!(
+    param_array::AbstractArray{T},
+    attributes::TimeSeriesAttributes{U},
+    ::Type{V},
+    model::DecisionModel,
+    ::DatasetContainer{DataFrameDataset},
+) where {
+    T <: Union{PJ.ParameterRef, Float64},
+    U <: PSY.AbstractDeterministic,
+    V <: PSY.HybridSystem,
+}
+    initial_forecast_time = get_current_time(model) # Function not well defined for DecisionModels
+    horizon = get_time_steps(get_optimization_container(model))[end]
+    components = get_available_components(V, get_system(model))
+    for component in components
```

Comment:
we probably need to loop over the subcomponents here instead of just the Hybrid.